test/mocks/router/mocks.h

```diff
@@ -362,5 +363,13 @@ class MockRouteConfigProviderManager : public RouteConfigProviderManager {
                                       Server::Configuration::FactoryContext& factory_context));
 };
 
+class MockScopedConfig : public ScopedConfig {
+public:
+  MockScopedConfig() = default;
```

Comment:
The empty defs for these should be in `.cc` for build performance reasons.

Reply:
Done.